MzTab_MTD

```diff
@@ -168,10 +227,322 @@ CREATE TABLE "MzTab"."PSM Section"
+    retention_time_in_seconds_end double precision,
+    abundance_assay_number text COLLATE pg_catalog."default",
+    abundance_assay text COLLATE pg_catalog."default",
+    CONSTRAINT "Small Molecule Feature (SMF) Section_pkey1" PRIMARY KEY ("SMF_ID"),
+    CONSTRAINT "Small Molecule Feature (SMF) Section_SME_ID_REFS_fkey" FOREIGN KEY ("SME_ID_REFS")
+        REFERENCES "MzTab-M"."Small Molecule Evidence (SME) Section" ("SME_ID") MATCH SIMPLE
+        ON UPDATE NO ACTION
+        ON DELETE NO ACTION
+        NOT VALID
+)
+
+TABLESPACE pg_default;
+
+ALTER TABLE "MzTab-M"."Small Molecule Feature (SMF) Section"
+    OWNER to postgres;
+
+-- Table: MzTab-M.Small Molecule Evidence (SME) Section
+
+-- DROP TABLE "MzTab-M"."Small Molecule Evidence (SME) Section";
+
+CREATE TABLE "MzTab-M"."Small Molecule Evidence (SME) Section"
+(
+    "SME_ID" integer NOT NULL,
+    evidence_input_id text COLLATE pg_catalog."default" NOT NULL,
+    database_identifier text COLLATE pg_catalog."default",
+    chemical_formula text COLLATE pg_catalog."default",
+    smiles text COLLATE pg_catalog."default",
+    inchi text COLLATE pg_catalog."default",
+    chemical_name text COLLATE pg_catalog."default",
+    uri text COLLATE pg_catalog."default",
+    derivatized_form text COLLATE pg_catalog."default",
+    adduct_ion text COLLATE pg_catalog."default",
+    exp_mass_to_charge double precision NOT NULL,
+    charge integer NOT NULL,
+    theoretical_mass_to_charge double precision NOT NULL,
+    spectra_ref text[] COLLATE pg_catalog."default" NOT NULL,
+    identification_method text COLLATE pg_catalog."default" NOT NULL,
+    ms_level text COLLATE pg_catalog."default" NOT NULL,
+    id_confidence_measure_number text COLLATE pg_catalog."default",
+    id_confidence_measure double precision,
+    rank integer NOT NULL,
+    CONSTRAINT "Small Molecule Evidence (SME) Section_pkey1" PRIMARY KEY ("SME_ID")
+)
+
+TABLESPACE pg_default;
+
+ALTER TABLE "MzTab-M"."Small Molecule Evidence (SME) Section"
+    OWNER to postgres;
+    
+    
+-- Table: XML.TraML
+
+-- DROP TABLE "XML"."TraML";
+
+CREATE TABLE "XML"."TraML"
+(
+    "id " text COLLATE pg_catalog."default",
+    "version " text COLLATE pg_catalog."default" NOT NULL,
+    "cvList" text[] COLLATE pg_catalog."default",
+    "SourceFileList" text[] COLLATE pg_catalog."default",
+    "ContactList" text[] COLLATE pg_catalog."default",
+    "PublicationList" text[] COLLATE pg_catalog."default",
+    "InstrumentList" text[] COLLATE pg_catalog."default",
+    "SoftwareList" text[] COLLATE pg_catalog."default",
+    "ProteinList" text[] COLLATE pg_catalog."default",
+    "CompoundList" text[] COLLATE pg_catalog."default",
+    "TransitionList" text[] COLLATE pg_catalog."default",
+    "TargetList" text[] COLLATE pg_catalog."default"
+)
+
+TABLESPACE pg_default;
+
+ALTER TABLE "XML"."TraML"
+    OWNER to postgres;
+    
+  
+ 
+-- Table: XML.TraML
+
+-- DROP TABLE "XML"."TraML";
+
+CREATE TABLE "XML"."TraML"
```

Comment:
What is the relation between this "TraML" table and the one above?

Reply:
i forgot to remove the wrong version of TraML (one above) , now i removed it